tools/linter/adapters/pip_init.py

```diff
@@ -52,11 +52,13 @@ def run_command(args: list[str]) -> subprocess.CompletedProcess[bytes]:
         stream=sys.stderr,
     )
 
-    uv_available = (
-        any(prefix in sys.base_prefix for prefix in ["uv/python", "uv\\python"])
-        and shutil.which("uv") is not None
-    )
+    uv_installed = shutil.which("uv") is not None
+    uv_python = any(prefix in sys.base_prefix for prefix in ["uv/python", "uv\\python"])
```

Comment:
I left the old check in for now, but frankly I'm suspecting the better option would be to only use uv install if we're in a venv.